boards/cogip-board/include/board.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2024 COGIP Robotics association
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_cogip-board COGIP 2023 Pegasus board
+ * @ingroup     boards_cogip-board
+ * @brief       Support for the COGIP 2023 Pegasus board
```

Comment:
2024

Pegasus is the platform name, the board name.

Reply:
done